src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import java.util.Map;
+
+public class FruitDaoImpl implements FruitDao {
+
+    @Override
+    public void addFruit(String fruit, Integer quantity) {
+        Storage.fruitStorage.put(fruit, quantity);
```

Comment:
When we have the method addSomething() we expect increasing quantity. But here we can put less fruits than it was before in the storage. You need to rename it or change it logic (and add another method for sell operation).